packages/core/src/components/carousel/stories/carousel.args.ts

```diff
@@ -1,161 +1,78 @@
+    hasNavigation: {
+      control: 'boolean',
+      description: 'If `true`, the carousel will have navigation buttons.',
+      defaultValue: true,
       table: {
         category: Category.PROPERTIES,
       },
     },
-    freeMode: {
-      description: 'Set to `true` to enable free mode functionality',
-      defaultValue: { summary: false },
+    hasPagination: {
+      control: 'boolean',
+      description: 'If `true`, the carousel will have pagination buttons.',
+      defaultValue: true,
       table: {
         category: Category.PROPERTIES,
       },
     },
     loop: {
-      description: ' Set to `true` to enable continuous loop mode',
-      defaultValue: { summary: false },
-      table: {
-        category: Category.PROPERTIES,
-      },
-    },
-    navigation: {
-      description: 'If `true`, enable pagination.',
-      defaultValue: { summary: true },
-      table: {
-        category: Category.PROPERTIES,
-      },
-    },
-    pagination: {
-      description: 'If `true`, enable pagination.',
-      defaultValue: { summary: true },
-      table: {
-        category: Category.PROPERTIES,
-      },
-    },
-    paginationClickable: {
-      description:
-        'If `true` then clicking on pagination button will cause transition to appropriate slide. Only for bullets pagination type.',
-      defaultValue: { summary: false },
-      table: {
-        category: Category.PROPERTIES,
-      },
-    },
-    paginationType: {
-      options: ['bullets', 'fraction', 'progressbar', 'custom', 'thumbnails'],
-      description:
-        ' String with type of pagination. Types: `bullets`, `fraction`, `thumbnails`, `progressbar` or `custom`.',
-      defaultValue: { summary: 'bullets' },
-      table: {
-        category: Category.PROPERTIES,
-      },
-    },
-
-    slidesPerGroup: {
-      description:
-        'Set numbers of slides to define and enable group sliding. Useful to use with slidesPerView > 1',
-      defaultValue: { summary: 1 },
-      table: {
-        category: Category.PROPERTIES,
-      },
-    },
-    slidesPerView: {
-      description:
-        "Number of slides per view (slides visible at the same time on slider's container).",
-      defaultValue: { summary: 1 },
-      table: {
-        category: Category.PROPERTIES,
-      },
-    },
-    spaceBetween: {
-      description: 'Distance between slides in px.',
-      defaultValue: { summary: 0 },
-      table: {
-        category: Category.PROPERTIES,
-      },
-    },
-    speed: {
-      description: 'Duration of transition between slides (in ms)',
-      defaultValue: { summary: 300 },
-      table: {
-        category: Category.PROPERTIES,
-      },
-    },
-    thumbnailImages: {
-      description:
-        'String of images URLs to be used as thumbnails splited by comma',
-      defaultValue: { summary: '' },
+      control: 'boolean',
+      description: 'If `true`, the user cannot interact with the button.',
```

Comment:
I think we can be clearer here. With navigation buttons or pagination buttons?

Reply:
Perfect, I improve it here [99b6924](https://github.com/juntossomosmais/atomium/pull/434/commits/99b69242a87e924e5af1311e9b416b827953a876)